dts/arm/nordic/nrf54h20_cpuapp.dtsi

```diff
@@ -25,6 +25,12 @@ wdt011: &cpuapp_wdt011 {};
 		compatible = "simple-bus";
 		interrupt-parent = <&cpuapp_nvic>;
 		ranges;
+
+		stmesp: memory@a2000000 {
+			compatible = "nordic,owned-memory";
```

Comment:
These memory regions shall not be described in UICR, so the `compatible` should be something else, like `reserved-memory`.
The SecDomFw shall configure MPC200 according to its internal rules. MPC110 does not require any configuration for STM to work. @hakonfam, is this implemented?

Reply:
i don't see `reserved-memory` compatible.

Reply:
Right, `/reserved-memory` is not a compatible, but a node name. Subnodes of `/reserved-memory` don't need a `compatible` property. So I guess you can remove from this node `compatible` and other properties connected with `nordic-owned-memory`.

Reply:
That is correct we configure the mpc200 entries for the stmesp address ranges outside of uicr mem handling now and don't want them in uicr anymore. We do the same for other mpc overrides where we have hardcoded entries that are just always valid and not part of a user configuration

Reply:
what about creating binding `arm,stmesp` (with just base.yaml) and using it as compatible. There is already `arm,itm` in `dts/bindings/debug`.

Reply:
If `stmesp` stands for STM e... stimulus ports then it would be even better solution.